### ISSUE TITLE
The Wizard random event spell may be causing crashes

### ISSUE THREAD

**Issue by Jalleo** [tgstation/-tg-station#5974]
This is probably down to memory Byond internally but whenever something to do with players occur the server lags a lot (with this spell that is). It may be causing crashes when other high memory intensive stuff occurs aswell.
@Incoming5643 
I know this is so because a crash just occured before this issue was created I was playing and whenever interactions with the players occured lag and lots of it

**Comment by MrStonedOne** [comment]
this can be caused by the rpg loot event and the petsposion event.

The first one increases memory by changing the name var, byond only stores changed vars per object, redirecting reads to initial(), It also changes the name, causing additional entries in the string tree.

The other one is making too many mobs from the petspolsion event, but this was mitigated by limiting its run count.

This has mostly been mitigated by a bug fix in byond in 508 relating to memory leaks, as the first one isn't enough to bring the memory count up too high on it's own.

You could close this issue.

**Comment by tkdrg** [comment]
tldr close this
